app/src/main/java/org/sopt/dosopttemplate/feature/home/HomeScreen.kt

```diff
@@ -0,0 +1,101 @@
+package org.sopt.dosopttemplate.feature.home
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Person
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.navigation.NavController
+import org.sopt.dosopttemplate.MainViewModel
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun HomeScreen(
+    modifier: Modifier = Modifier,
+    navController: NavController,
+    viewModel: MainViewModel,
+) {
+    Scaffold { innerPadding ->
+        HomeContent(
+            modifier = modifier.padding(innerPadding),
+            navController = navController,
+            viewModel = viewModel,
+        )
+    }
+}
+
+@Composable
+fun HomeContent(
+    modifier: Modifier,
+    navController: NavController,
+    viewModel: MainViewModel,
+) {
+    val state = viewModel.state.collectAsState()
+    Column(
+        modifier = modifier
+            .padding(horizontal = 16.dp)
+            .padding(top = 32.dp),
+    ) {
+        Row(
+            verticalAlignment = Alignment.CenterVertically,
+        ) {
+            Icon(
+                imageVector = Icons.Default.Person,
+                contentDescription = "Person Icon",
+            )
+            Spacer(modifier = Modifier.size(8.dp))
+            Text(
+                text = state.value.nickname,
```

Comment:
```suggestion
                text = state.nickname,
```

collect를 할 때 by 문법을 활용하면 바로 value에 접근할 수 있습니다.